ogn/parser/parse.py

```diff
@@ -48,7 +56,8 @@ def parse_ogn_aircraft_beacon(aprs_comment):
                 'gps_status': ac_match.group('gps_accuracy'),
                 'software_version': float(ac_match.group('flarm_software_version')) if ac_match.group('flarm_software_version') else None,
                 'hardware_version': int(ac_match.group('flarm_hardware_version'), 16) if ac_match.group('flarm_hardware_version') else None,
-                'real_address': ac_match.group('flarm_id')}
+                'real_address': ac_match.group('flarm_id'),
+                'power': float(ac_match.group('power')) if ac_match.group('power') else None}
```

Comment:
I've been thinking about this for my Ruby implementation as well, so here are my 5 cents: How about renaming `signal` to `signal_quality` (aka: signal to noise ratio) and `power` to `signal_power`?